File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/swift/F09.P1.swift
func probe(_ s1: String, _ s2: String) -> Bool {
// BEGIN PROBE F09.P1
    let eq = s1 == s2
// END PROBE F09.P1
    return eq
}

let base = "abcdefghij"
print(probe(base + "klmnopqrst", base + "klmnopqrst"))

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/cpp/F09.P1.cpp
#include <cstdio>
#include <string>

int main() {
    std::string base = "ab";
    std::string s1 = base + "cd";
    std::string s2 = base + "cd";
// BEGIN PROBE F09.P1
    bool eq = s1 == s2;
// END PROBE F09.P1
    std::printf("%d\n", eq);
}
